File: 250721/약수가 세 개인 수/numbers-with-three-divisors.py
start, end = map(int, input().split())
total=0
for i in range(start, end+1):
    sum=0
    for j in range(1,i+1):
        if i%j==0:
            sum+=1
    if sum==3:
        total+=1
print(total)
# Please write your code here.

File: 260126/약수가 세 개인 수/numbers-with-three-divisors.cpp
#include <iostream>

using namespace std;

int st, ed;

int main() {
    cin >> st >> ed;
    int total=0;
    for(int i=st;i<=ed;i++){
        int num=0;
        for(int j=1;j<=i;j++){
            if(i%j==0){
                num+=1;
            }
        }
        if(num==3){
            total+=1;
        }
    }
    cout<<total;
    // Please write your code here.

    return 0;
}
